src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -197,7 +197,7 @@ private static bool IsErrorStatus(SecurityStatusPalErrorCode errorCode)
             }
 
             SNIHandle sniHandle = null;
-            switch (details._connectionProtocol)
+            switch (details.ResolvedProtocol)
```

Comment:
This change has already been reviewed via the other PR. Can you merge that and then rebase/merge this branch with feat/sqlclientx ?

Reply:
Yep, this PR has been updated to include those merged changes now